packages/platform-android/native_modules.gradle

```diff
@@ -183,15 +183,20 @@ class ReactNativeModules {
     if (this.reactNativeModules != null) return this.reactNativeModules
 
     ArrayList<HashMap<String, String>> reactNativeModules = new ArrayList<HashMap<String, String>>()
-    def npx = Os.isFamily(Os.FAMILY_WINDOWS) ? "npx.cmd" : "npx"
-    def command = "${npx} --quiet --no-install react-native config"
+
     /**
-     * Running npx from the directory of the JS app which holds this script in its node_modules.
-     * We do so, because Gradle may be ran with a different directory as CWD, that's outside of JS project
-     * (e.g. when running with -p flag), in which case npx wouldn't resolve correct `react-native` binary.
+     * Resolve the CLI location from Gradle file
+     *
+     * @todo: Sometimes Gradle can be called outside of the JavaScript hierarchy (-p flag) which
+     * will fail to resolve the script and the dependencies. We should resolve this soon.
+     *
+     * @todo: `fastlane` has been reported to not work too.
```

Comment:
I have managed to confirm one additional place where it fails and I am hoping this will let us better debug this.